charts/kubearchive/templates/operator/operator.yaml

```diff
@@ -48,34 +48,59 @@ spec:
             capabilities:
               drop:
                 - ALL
-        - args:
+        - name: manager
+          {{- if .Values.operator.debug }}
+          args:
+            - --listen=:40000
+            - --headless=true
+            - --accept-multiclient
+            - --api-version=2
+            - --log
+            - exec
+            - /ko-app/operator
+            - --
+            - --health-probe-bind-address=:8081
+            - --metrics-bind-address=127.0.0.1:8080
+            - --leader-elect
+          command:
+            - ./go/bin/dlv
+          {{- else }}
+          args:
             - --health-probe-bind-address=:8081
             - --metrics-bind-address=127.0.0.1:8080
             - --leader-elect
           command:
             - /ko-app/operator
+          {{- end}}
```

Comment:
```suggestion
          {{- end }}
```

Reply:
It doesn't apply anymore now that this is provided by the `--debug` option of the `ko build` command